src/PolyhedralGeometry/Polyhedron/properties.jl

```diff
@@ -413,7 +416,7 @@ julia> lattice_volume(C)
 8
 ```
 """
-lattice_volume(P::Polyhedron) = (pm_object(P)).LATTICE_VOLUME
+lattice_volume(P::Polyhedron) = (pm_object(P)).LATTICE_VOLUME::Int
```

Comment:
This can become very large and is always a `Polymake.Integer`, so this should be converted to some big integer type:
```suggestion
lattice_volume(P::Polyhedron)::fmpz = (pm_object(P)).LATTICE_VOLUME
```